packages/project-access/src/constants.ts

```diff
@@ -1,7 +1,9 @@
 export const FileName = {
+    Fragment: 'fragment.xml',
```

Comment:
Are these really file names and not suffices that maybe should go to a different object?